public/html.css

```diff
@@ -154,3 +154,15 @@ sub {
   bottom: -.25em;
 }
 
+
+@media (prefers-color-scheme: dark) {
+  body{
+    background-color:#202020;
+    color:#eee  ;
```

Comment:
Here there are too many spaces, you should remove any space before `;` to fit with the rest of the file.

Make sure you apply these rules to all the CSS you changed.